src/annotator/test/adder-test.js

```diff
@@ -22,115 +22,110 @@ function revertOffsetElement(el) {
   el.style.top = '0';
 }
 
-describe('annotator.adder', function() {
+// nb. These tests currently cover the `AdderToolbar` Preact component as well
+// as the `Adder` container.
```

Comment:
I assume this is for a good (or expedient) reason. Out of curiosity, what is that reason? Is `AdderToolbar` not independent enough to test on its own? Something else? Do you foresee this getting less coupled as we go?

Reply:
Oh, I forgot to mention. This was to prevent the diff from being too large and to show that there were no functional changes in this commit. I plan to separate them in future.